test/tests.js

```diff
@@ -241,8 +241,29 @@ QUnit.test('false secure value', function (assert) {
 	assert.strictEqual(actual, expected, 'false should not modify path in cookie string');
 });
 
+QUnit.test('unofficial attribute', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: 'anything'
+	}), 'c=v; path=/; unofficial=anything', 'should write the cookie string with unofficial attribute');
+});
+
+QUnit.test('unofficial attribute with boolean value', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: true
```

Comment:
Another good thing about cutting scope is that the PR becomes simpler so it's easy to focus on what really matters, which is the `SameSite` for now.